containers/proxy-httpd-image/uyuni-configure.py

```diff
@@ -124,17 +124,58 @@ def getIPs(fqdn: str) -> Tuple[str, str]:
         if len(proxyIPv6) > 0:
            requireIPv6 = "Require ip {}".format(proxyIPv6)
         file.write(f'''<Directory "/srv/www/tftpsync">
-    <IfVersion >= 2.4>
-        <RequireAny>
-            {requireIPv4}
-            {requireIPv6}
-        </RequireAny>
-    </IfVersion>
+    <RequireAny>
+        {requireIPv4}
+        {requireIPv6}
+    </RequireAny>
 </Directory>
 
 WSGIScriptAlias /tftpsync/add /srv/www/tftpsync/add
 WSGIScriptAlias /tftpsync/delete /srv/www/tftpsync/delete''')
 
+    with open("/etc/apache2/conf.d/saltboot.conf", "w") as file:
+        # Saltboot uses the same URL regardles containerized or normal proxy
+        # here we rewrite URL so upstream server understands it
+
+        # First condition/rule removes double arch rule for backward compatility
+        # second changes /saltboot/image/..?orgid=<org> to /os-images/<org>/
+        file.write('''
+RewriteEngine on
```

Comment:
Could you please fix indentation here?

Reply:
I do not follow, what indentation problem is here?

Reply:
I mean I'd prefer if the string file content starts being aligned with the space indentation of the line above 142, the same like [here](https://github.com/uyuni-project/uyuni/blob/master/containers/proxy-httpd-image/uyuni-configure.py#L87). It's probably personal feeling, we don't have rules about it.

Reply:
Ah, I did it like this because before I was asked to do so ( see https://github.com/uyuni-project/uyuni/pull/5089#discussion_r836103208 )

Reply:
Yeah @ncounter I would personally not prefer to have it the way you linked since this will reduce the readability of the generated file.

Reply:
Np, I'm okay with that either. Orthogonal question: do we have checkstyle or something in the python environment?

@aaannz drop my comments about it then

Reply:
> do we have checkstyle or something in the python environment?

Currently not. I'm working on setting up pylint on the side. If it works out as planned, we'll use https://github.com/mbarkhau/pylint-ignore to not overwhelm us with existing linting errors/warnings.